storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -1023,6 +1025,13 @@ struct my_option xb_client_options[] =
     "privileges before performing any query.", &opt_check_privileges,
    &opt_check_privileges, 0, GET_BOOL, NO_ARG, 0, 0, 0, 0, 0, 0},
 
+  {"datafile_buffer_size_multiplicator",
+   OPT_XTRA_DATAFILE_READ_BUFFER_MULTIPLICATOR,
```

Comment:
I would prefer to specify read buffer size in bytes. Using multiplier feels confusing. Xtrabackup could divide it by page size. It is also possible to restrict this parameter so that it will be multiple of 16k. Maybe something like `read-buffer-size`?